.golangci.yml

```diff
@@ -97,6 +97,7 @@ linters:
     - varcheck  # deprecated
     - unused  # deprecated
     - goimports
+    - dupl
```

Comment:
Required to be turned off, until we decede to unify Connection and NetworkService protocols.

Reply:
That's OK